manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BethesdaSoftworks.Bethesda
+PackageVersion: 1.75.0.0
+PackageLocale: en-US
+Publisher: Bethesda Softworks
+PackageName: Bethesda.net Launcher
+License: Microsoft
+ShortDescription: Bethesda.net Launcher Setup

```

Comment:
Bethesda Launcher is a digital distributor exclusively for games developed by Bethesda Softworks